go.mod

```diff
@@ -1,27 +1,60 @@
 module github.com/zaquestion/lab
 
 require (
+	github.com/DATA-DOG/go-sqlmock v1.3.3 // indirect
+	github.com/GeertJohan/go.rice v1.0.0 // indirect
+	github.com/Microsoft/go-winio v0.4.15 // indirect
 	github.com/acarl005/stripansi v0.0.0-20180116102854-5a71ef0e047d
-	github.com/araddon/dateparse v0.0.0-20200409225146-d820a6159ab1
-	github.com/avast/retry-go v0.0.0-20180319101611-5469272a8171
-	github.com/charmbracelet/glamour v0.1.1-0.20200114010931-28cbdae8e7a9
-	github.com/dlclark/regexp2 v1.2.0 // indirect
-	github.com/fatih/color v1.9.0
-	github.com/gdamore/tcell v1.3.0
-	github.com/go-git/go-git/v5 v5.0.0
-	github.com/lunixbochs/vtclean v0.0.0-20180621232353-2d01aacdc34a
+	github.com/alecthomas/chroma v0.8.1 // indirect
+	github.com/alecthomas/kong-hcl v0.1.8-0.20190615233001-b21fea9723c8 // indirect
+	github.com/araddon/dateparse v0.0.0-20201001162425-8aadafed4dc4
+	github.com/avast/retry-go v3.0.0+incompatible
+	github.com/charmbracelet/glamour v0.2.0
+	github.com/dlclark/regexp2 v1.4.0 // indirect
+	github.com/fatih/color v1.10.0
+	github.com/fsnotify/fsnotify v1.4.9 // indirect
+	github.com/gdamore/tcell v1.4.0
+	github.com/gdamore/tcell/v2 v2.0.1-0.20201017141208-acf90d56d591
+	github.com/go-git/go-git/v5 v5.2.0
+	github.com/golang/protobuf v1.4.3 // indirect
+	github.com/google/goterm v0.0.0-20200907032337-555d40f16ae2 // indirect
+	github.com/gorilla/csrf v1.6.0 // indirect
+	github.com/gorilla/handlers v1.4.1 // indirect
+	github.com/gorilla/mux v1.7.3 // indirect
+	github.com/hashicorp/go-retryablehttp v0.6.8 // indirect
+	github.com/imdario/mergo v0.3.11 // indirect
+	github.com/kevinburke/ssh_config v0.0.0-20201106050909-4977a11b4351 // indirect
+	github.com/lunixbochs/vtclean v1.0.0
+	github.com/magiconair/properties v1.8.4 // indirect
+	github.com/microcosm-cc/bluemonday v1.0.4 // indirect
+	github.com/mitchellh/mapstructure v1.3.3 // indirect
+	github.com/muesli/reflow v0.2.0 // indirect
+	github.com/muesli/termenv v0.7.4 // indirect
 	github.com/onsi/gomega v1.4.3 // indirect
-	github.com/pkg/errors v0.8.1
-	github.com/rivo/tview v0.0.0-20191129065140-82b05c9fb329
-	github.com/rsteube/carapace v0.0.16
-	github.com/spf13/afero v1.1.2
-	github.com/spf13/cobra v1.0.0
+	github.com/pelletier/go-toml v1.8.1 // indirect
+	github.com/pkg/errors v0.9.1
+	github.com/rivo/tview v0.0.0-20201018122409-d551c850a743
+	github.com/rsteube/carapace v0.1.6
+	github.com/russross/blackfriday/v2 v2.1.0 // indirect
+	github.com/spf13/afero v1.4.1
+	github.com/spf13/cast v1.3.1 // indirect
+	github.com/spf13/cobra v1.1.1
+	github.com/spf13/jwalterweatherman v1.1.0 // indirect
 	github.com/spf13/pflag v1.0.5
-	github.com/spf13/viper v1.4.0
-	github.com/stretchr/testify v1.4.0
+	github.com/spf13/viper v1.7.1
+	github.com/stretchr/testify v1.6.1
 	github.com/tcnksm/go-gitconfig v0.1.2
-	github.com/xanzy/go-gitlab v0.33.1-0.20200713191942-71ea998bed24
-	golang.org/x/crypto v0.0.0-20200302210943-78000ba7a073
+	github.com/xanzy/go-gitlab v0.39.0
```

Comment:
Nice. With this update, MR #450 may get concluded.

Reply:
@bmeneguele can you test #450 on top of my changes?  Or do you just want me to merge and then you can deal with #450?

Reply:
@prarit you can merge this one and I deal with #450 later.